Dockerfile

```diff
@@ -211,6 +211,7 @@ RUN pip install mpld3 && \
     pip install CVXcanon && \
     pip install fancyimpute && \
     pip install pymc3 && \
+    pip install imagecodecs && \
```

Comment:
Can you add a test reading a multi-image file to prevent regression? Thank you

The easiest way is to wait until Jenkins has built your image:
https://ci.kaggle.net/job/kernels/job/docker-python/job/add_imagecodecs/

The image will have the following tag `gcr.io/kaggle-images/python:add_imagecodecs-pretest`.

You can add the test to the `test_skimage.py`: https://github.com/Kaggle/docker-python/blob/master/tests/test_skimage.py

You can upload the image you need here: https://github.com/Kaggle/docker-python/tree/master/tests/data

The files in the `/tests/data` directory are accessible to the test at `/input/tests/data/your_file.your_extension`. Example: https://github.com/Kaggle/docker-python/blob/33ca5906ac82bf331b0d47b506cb741b1398d0a1/tests/test_opencv.py#L7

You can then iterate on your test using:

```
cd path/to/this/repo/docker-python
sudo ./test -i gcr.io/kaggle-images/python:add_imagecodecs-pretest -p test_skimage.py
```